example.json

```diff
@@ -37,7 +37,10 @@
       }
     },
     "fruit": {
-      "type": "object"
+      "type": "object",
+      "additionalProperties": {
+        "type": "string"
+      }
```

Comment:
what's the idea here?

Reply:
The tests in `cargo-typify/tests/integration.rs` rely on this, and I wanted to ensure that they would generate the correct map when prompted. The previous `example.json` file would only generate a `serde_json::Map` due to `T` not being defined.

Reply:
This allowed me to validate generating, e.g.  a BTree instead. :)